Libraries/Components/WebView/WebView.ios.js

```diff
@@ -462,6 +462,16 @@ var WebView = React.createClass({
   },
 
   /**
+   * Evaluate JavaScript on the current page.
+   */
+  evaluateJavaScript: async function(script: string): Promise {
+    var escaped = JSON.stringify(script).replace(/\u2028/g, '\\u2028').replace(/\u2029/g, '\\u2029');
+    var wrapped = 'JSON.stringify(eval(' + escaped + '))';
```

Comment:
Why enforce this `JSON.stringify(eval(` pattern? Why not just send over the original string? If your return value isn't string representable, you should add the JSON.stringify yourself.